### ISSUE TITLE
It does not work with kodi 17.3

### ISSUE THREAD

**Issue by tmontanaro** [HIGHWAY99/repository.thehighway#24]
It does not work with kodi 17.3

I think the problem is simple: it seems that the list of playing videos is shown in a part of the screen that is not visible!!!

I tried to use it with
- streamondemand 5.5.8 (https://github.com/streamondemand/plugin.video.streamondemand/releases)
- Cypher TV

**Comment by prahjister** [comment]
Will this be made to work on kodi 17?

**Comment by Alistair1231** [comment]
try a different skin. for me it dowsn't work with default, but with confluence or xonfire or something like that it works...

**Comment by kamalt006** [comment]
yes it works in the skins other than default skin in kodi 17.3 or above

**Comment by tmontanaro** [comment]
In the same day in which I wrote the message, I updated Kodi (to 17.4) so I cannot try to apply your suggestion anymore! I hope someone else can try it to provide the final solution and, consequently, to close the issue!
